jira.json

```
@@ -16,12 +16,12 @@
     "product_vendor": "Atlassian",
     "product_name": "Jira",
     "product_version_regex": ".*",
-    "min_phantom_version": "6.3.0",
+    "min_phantom_version": "7.0.0",
     "latest_tested_versions": [
         "On prem v8.21.0",
-        "Cloud Copyright (c) 2002 - 2023 Atlassian Corporation Pty Ltd."
+        "Cloud Copyright (c) 2002 - 2025 Atlassian Corporation Pty Ltd."
     ],
-    "python_version": "3",
+    "python_version": "3.9, 3.13",
```

Since this app will no longer work with 3.9, we must set the python version to 3.13 only
```suggestion
    "python_version": "3.13",
```

---

Done